server/scraper.py

```diff
@@ -0,0 +1,59 @@
+import re
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.firefox.options import Options
+
+PATH = '/home/user/scraping/geckodriver'
```

Comment:
Rather than manually installing the driver, let's use a library to do that for us: https://pypi.org/project/webdriver-manager/